tests/core/daemon/test_daemon.py

```diff
@@ -60,6 +62,9 @@ class Daemon:
     services: Dict[str, Union[List[Service], Service]]
     connections: Dict[str, Optional[List[Any]]]
 
+    # Instance variables used by WebSocketServer.get_wallet_addresses()
+    net_config: Dict[str, Any]
```

Comment:
Maybe this would be nice?

```suggestion
    net_config: Dict[str, Any] =  field(default_factory=dict)
```

Reply:
Updated